src/clients/memberclient.cpp

```diff
@@ -122,6 +125,22 @@ void submit_accept_node(RpcTlsClient& tls_connection, NodeId node_id)
   cout << response.dump() << endl;
 }
 
+NodeId submit_add_node(RpcTlsClient& tls_connection, NodeInfo& node_info)
```

Comment:
I think it's probably simpler to add new admin rpcs to the Python client instead, quicker/less code. But this is fine too, of course.